index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Word counter</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<script src="index.js"></script>
+<div id="main">
+    <h2>Word counter</h2>
+
```

Comment:
Please pay attention to the code formatting.
(this needs to be corrected in all places and in all files)